### PR TITLE
Minor grammar corrections

### PR DESCRIPTION
**Pull request by hashworks**

### CODE REVIEW

**Comment by kenyon** [comment]
IMO it should be left as “catalog” and other American spellings, since that’s what the official Puppet docs and commands use, as a US company.

File: README.md [hunk 1]
```diff
@@ -21,16 +21,16 @@ Puppet module to configure wireguard through systemd-networkd configs
 ## Setup
 
 The module can create firewall rules with [voxpupuli/ferm](https://github.com/voxpupuli/puppet-ferm#puppet-ferm).
-This is enabled by default can be disabled by setting the `manage_firewall`
+This is enabled by default but can be disabled by setting the `manage_firewall`
 parameter to false in the `wireguard::interface` defined resource. You need to
-have the `ferm` class in your catalog to use the feature.
+have the `ferm` class in your catalogue to use the feature.
```
Comment (bastelfreak) [inline README.md:26]: I'm fine with the changes except for catalog->catalogue, because catalog is the official term in the Puppet docs.

**Comment by root-expert** [comment]
License and and Licence is also a US/UK thingy :smile:

**Comment by hashworks** [comment]
> License and and Licence is also a US/UK thingy smile

Well, I asked @bastelfreak if he wanted US or UK ;)

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**